app/mailers/pom_mailer.rb

```diff
@@ -22,71 +22,40 @@ def new_allocation_email(params = {})
     mail(to: params[:pom_email])
   end
 
-  def responsibility_override(
-    message:,
-    prisoner_number:,
-    prisoner_name:,
-    prison_name:,
-    email:
-  )
+  def responsibility_override
     set_template('ca952ba5-58b5-4e2d-8d87-60590d76560c')
-    set_personalisation(prisoner_name: prisoner_name,
-                        prisoner_number: prisoner_number,
-                        reason: message,
-                        prison_name: prison_name)
-    mail(to: email)
+    set_personalisation(**params.slice(:prisoner_name, :prisoner_number, :reason, :prison_name))
```

Comment:
I don't think `reason` exists in the params does it? In `responsibilities_controller` it seems to be called `message`

Reply:
Nice catch!!! Bad tests...

Reply:
Fiaxed